lib/src/api/opt/mod.rs

```diff
@@ -349,7 +349,7 @@ fn into_json(value: Value, simplify: bool) -> JsonValue {
 			false => json!(d),
 		},
 		Value::Datetime(d) => d.to_raw().into(),
-		Value::Uuid(uuid) => json!(uuid),
+		Value::Uuid(uuid) => json!(uuid.0),
```

Comment:
@rushmorem, with regards to *just* this line change, is this needed, and does this fix anything for `sql::Uuid`?

Reply:
Yes @tobiemh, I think it's needed. I did the same in https://github.com/surrealdb/surrealdb/pull/2254.